web/app_options.js

```diff
@@ -236,6 +236,12 @@ const defaultOptions = {
     value: false,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  enableAutolinking: {
+    // TODO: remove it when unnecessary.
+    /** @type {boolean} */
+    value: false,
+    kind: OptionKind.VIEWER,
+  },
```

Comment:
- This isn't sorted alphabetically (and please fix the casing).
 - I don't think this should be temporary, since even Adobe Reader allows this to be configured by the user. (Personally I really want to be able to disable this feature.)
 - It should be a preference as well.

```suggestion
  enableAutoLinking: {
    /** @type {boolean} */
    value: false,
    kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
  },
```